ui/src/submissions/index.jsx

```diff
@@ -75,6 +78,11 @@ class Submissions extends Component {
               path={`${SUBMISSIONS_SEMINAR}/:id`}
               component={SeminarUpdateSubmissionPageContainer}
             />
+            <Route
```

Comment:
it should be a private route https://github.com/inspirehep/inspirehep/blob/ba5fe0fdd9aa4daebad985376068487edccf995a/ui/src/common/PrivateRoute.jsx#L10

Reply:
Changed to private route